packages/synthetic-chain/package.json

```diff
@@ -2,20 +2,17 @@
   "name": "@agoric/synthetic-chain",
   "version": "0.0.5-4",
   "description": "Utilities to build a chain and test proposals atop it",
-  "bin": "./cli.ts",
-  "main": "cli.ts",
+  "bin": "./dist/cli/cli.js",
+  "main": "./dist/cli/cli.js",
```

Comment:
Does it make sense to keep a `main` pointed at the CLI?

Reply:
I believe it does. This allows, e.g. `npx @agoric/synthetic-chain`. 

Why do you think it might not?

Reply:
my understanding is that `npx` uses `bin`. Main is only used for `require` (as such it may be entirely unnecessary)

Reply:
ah, I misread https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main

thanks.